hearts/tests/test_hearts.py

```diff
@@ -177,6 +177,12 @@ def test_lead_the_trick():
     assert round1.turn_counter == 0
     assert round1.hearts_broken is False
 
+    # Leading the trick with only hearts in hand
+    hand = Hand.deserialize(['2h', 'Jh', 'Ah'])
+    round1.hands[P1] = hand
+    round1.lead_the_trick(P1, Card('J', 'h'))
+    assert round1.hearts_broken is True
+
```

Comment:
Can you move this to its own test? And just test it from `play_card` instead of testing it from `lead_the_trick`